### PR TITLE
Fix scoop power consumption.

### PR DESCRIPTION
**Pull request by Greevv**
#### Summary
SUMMARY: Balance "Fixes absurdly high power usage of the scoop vehicle part."

#### Purpose of change
While trying out scoops for the first time ingame I noticed that my vehicle immediately lost all power as soon as I ran over some items. Some investigation in the code revealed that the code had not been updated since 2015 and the power usage was using the old vehicle power numbers. Scooping a 1kg item would use between 8000 and 15000 ePower. This change fixes it to a much more reasonable formula.

#### Describe the solution
I did some research of the power consumption of industrial vacuums and electric street sweepers. In every calculation I did the amount of ePower used per item came out to less than one outside of some weird edge cases. I created a new formula with this in mind that functions instead off of volume, as that was the only real world data I could reliably find. The 1 ePower per liter number is actually extremely generous but considering we can't use less than 1 ePower per action that was the number I had to go with. 

#### Describe alternatives you've considered
Considered going through old commits and trying to convert the old power consumption formula to the new power numbers, but this would take lots and lots of time and I figured a formula using real world data would be better than the old, which appeared to be pulled out of someone's ass.

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
Perhaps you could add a comment stating what the formula you used is, or explain it in the PR?

**Comment by Greevv** [comment]
The formula basically just makes the scoop consume 1 epower per liter of the item picked up, with a minimum power consumption of 1.

**Comment by anothersimulacrum** [comment]
Yes, but could you add the reasoning for why that is the formula, so that someone doesn't come along in a couple of years and say that it looks like the formula got "pulled out of someone's ass".

File: src/vehicle_use.cpp [hunk 1]
```diff
@@ -1149,9 +1149,12 @@ void vehicle::operate_scoop()
                 sounds::sound( position, rng( 10,
                                               that_item_there->volume() / units::legacy_volume_factor * 2 + 10 ),
                                sounds::sound_t::combat, _( "BEEEThump" ) );
+            } 
+            int scoopvol = units::to_milliliter<int>( that_item_there->volume() );
+            if (scoopvol < 1000) {
```
Comment (ZhilkinSerg) [inline src/vehicle_use.cpp]:
```suggestion
            if( scoopvol < 1000 ) {
```

**Comment by Greevv** [comment]
Added comments to the code explaining my rationale and used Serg's suggested formatting.

**Comment by kevingranade** [comment]
The mechanism here is almost certainly some kind of spinning roller. It wouldn't use power with each item, but rather use power every turn it's activated.

**Comment by Greevv** [comment]
It already does that, the per item cost was just in addition to it.

**Comment by Greevv** [comment]
Also, I think that it's a reasonable assumption that when it picks an item up it would increase the load on the motors of the roller, thereby increasing power consumption briefly.

**Comment by kevingranade** [comment]
Roughly speaking, this is going to be in the neighborhood of 20J per kg of the lifted item. One unit of epower is 1 kilojoule, so the contribution of each item is going to be minuscule. One unit of epower would lift about 50kg of items.
This is dwarfed by the overhead of simply operating the machine.

**Comment by Greevv** [comment]
Of course, I just didn't want to remove the code completely, but if you
feel that's what should be done, I can do that.

**Comment by Greevv** [comment]
Removed the per-item power drain and doubled the constant power drain to account for this.

**Pull request merged**